From a67f84998678ca6cf4e8d1135608d41d88e96e0c Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <cyril@debamax.com>
Date: Wed, 15 Apr 2026 05:09:55 +0200
Subject: [PATCH] Fix python program detection.

The relevant part got cleaned up in the most recent commits, but let's
go for incremental fixes.

Signed-off-by: Cyril Brulebois <cyril@debamax.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8103410..c36ff77 100644
--- a/subprojects/frida-tools/meson.build
+++ b/subprojects/frida-tools/meson.build
@@ -11,7 +11,7 @@ project(
 )
 
 meson.add_dist_script(
-  find_program('python'),
+  find_program('python3'),
   meson.current_source_dir() / 'set_dist_version.py',
   meson.project_version(),
 )
-- 
2.39.5

